modules/operator-upgrade-modify-central-custom-resource.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_module-type: PROCEDURE
+[id="operator-upgrade-modify-central-custom-resource_{context}"]
+= Modifying Central custom resource
+
+[role="_abstract"]
+If you have not configured a default storage class for {product-title-short} that is SSD or high performance, you must update the `Central` custom reource to request persistent storage by creating a persistent volume claim (PVC).
```

Comment:
maybe something like request persistent storage for the new Central DB service by...